app/controllers/DefaultsController.scala

```diff
@@ -61,7 +63,9 @@ class DefaultsController(val config: ApplicationConfiguration, val acl: Acl, val
           config.facia.navListType,
           Metadata.tags.map{
             case (_, meta) => meta
-          }
+          },
+          routes.FaciaContentApiProxy.capiLive("").absoluteURL(true),
```

Comment:
Is this going to know what the correct domain is?

Reply:
Yeah, any route builders inside the Play app will have to use something similar to this (or something similar) to build absolute URLs.